int8 doserMotorRealSpeed
bool doserMotorStatus
